crates/cxx-qt-build/src/lib.rs

```diff
@@ -498,6 +498,25 @@ impl CxxQtBuilder {
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..=qtbuild.version().minor {
+            let at_least_qt_version = format!(
+                "cxxqt_at_least_qt_version_{}_{}",
+                qtbuild.version().major,
+                minor
+            );
+            println!("cargo:rustc-cfg={}", at_least_qt_version);
+            let variable_cargo = format!("CARGO_CFG_{}", at_least_qt_version);
+            env::set_var(variable_cargo, "true");
+        }
+
+        // We don't support Qt < 5
+        for n in 5..=qtbuild.version().major {
```

Comment:
tiny nit, could use `major` instead of `n`

Reply:
suspect it'll make cargo fmt wrap the next line though :-) but it's easier to read